admin/includes/comment.php

```diff
@@ -1,27 +1,26 @@
-<?php //include("init.php");
+<?php
+// include("init.php");
+
 class Comment extends Db_object
 {
-
     protected static $db_table = "comments";
-    protected static $db_table_fields = array('photo_id', 'author', 'id', 'body');
+    protected static $db_table_fields = ['photo_id', 'author', 'id', 'body',];
     public $id;
     public $photo_id;
     public $author;
     public $body;
+
     public static function create_comment($photo_id, $author, $body)
```

Comment:
This function some times is void, it should always return something

Reply:
I've lost a curly bracket. Before committing test everything in the browser.

Reply:
This file has a violation of Symfony standard: `Add a single space around binary operators (==, &&, ...), with the exception of the concatenation (.) operator;`
Read carefully the constraints!!!